rust/rust.bzl

```diff
@@ -167,6 +188,19 @@ def _get_features_flags(features):
     features_flags += ["--cfg feature=\\\"%s\\\"" % feature]
   return features_flags
 
+def _get_crate_type_and_target_outputs(name, crate_type, platform):
+  extension = LIBRARY_AND_PLATFORM_TO_EXTENSION[crate_type or "rlib", platform or "linux"]
```

Comment:
This is done (and was done a while ago)

Hoping posting this comment will complete the review.